rules/scala_proto/private/core.bzl

```diff
@@ -33,6 +33,8 @@ def scala_proto_library_implementation(ctx):
     args = ctx.actions.args()
     args.add("--output_dir", gendir.path)
     args.add_all("--", transitive_sources)
+    args.set_param_file_format("multiline")
```

Comment:
What is this for?

Reply:
Bazel 0.27 started enforcing that workers have a flagfile. If this is not set, Bazel fails because it cannot detect a flagfile for the worker.